lib/validators/modelValidator.ts

```diff
@@ -675,14 +676,16 @@ export class ModelValidator extends SpecValidator<SpecValidationResult> {
     if (responseWithoutXmsExamples && responseWithoutXmsExamples.length) {
       const msg =
         `Following response status codes "${responseWithoutXmsExamples.toString()}" for ` +
-        `operation "${
-          operation.operationId
-        }" were present in the swagger spec, ` +
+        `operation "${operation.operationId}" were present in the swagger spec, ` +
         `however they were not present in x-ms-examples. Please provide them.`
       const e = this.constructErrorObject<Sway.ValidationEntry>(
         ErrorCodes.ResponseStatusCodeNotInExample,
         msg
       )
+      setPositionAndUrl(
+        e,
+        getTitle(operation.definition)
```

Comment:
so I guess because we manually construct this one the source mapping was not coming ?